docs/localnet/localnet.md

```diff
@@ -242,7 +251,7 @@ web_ui:
   port: 8082
 migration_web_ui:
   host: localhost
-  port: 8079
+  port: 8078
```

Comment:
curious to hear what collision happened here?

Reply:
This collided with the migration web UI of frodo.